sensing/pointcloud_preprocessor/src/outlier_filter/dual_return_outlier_filter_nodelet.cpp

```diff
@@ -111,12 +142,23 @@ void DualReturnOutlierFilterComponent::filter(
     vertical_bins);  // TODO(davidw): this is for Pandar 40 only, make dynamic
   weak_first_pcl_input_ring_array.resize(vertical_bins);
 
+  const uint ring_number = vertical_bins;
+  const uint azimuth_steps = 36000;
+  float * pcl_input_distance_array = new float[ring_number * azimuth_steps];
+  float * pcl_weak_first_distance_array = new float[ring_number * azimuth_steps];
```

Comment:
These variables start with "pcl" but I suppose they are not depends on PCL.
Remove pcl prefix.

Reply:
I agree. "pcl" is removed.